volttron/platform/web/admin_endpoints.py

```diff
@@ -184,8 +185,11 @@ def verify_and_dispatch(self, env, data):
             elif page == 'pending_auth_reqs.html':
                 self._pending_auths = self._rpc_caller.call(AUTH, 'get_authorization_failures').get()
```

Comment:
Probably should make this a single call rather than 3 different ones...

Reply:
done